docs/tutorial/index.html

```diff
@@ -17,6 +17,194 @@
+<h2 id="introduction">Introduction</h2>
+<p>Testo is a test framework for OCaml. Like with OUnit or Alcotest, the user writes a collection of tests. A test consists of a name and an OCaml test function to run, with some options. If the test function returns, the test is considered successful but if it raises an exception, it is considered failed.</p>
+<p>The test suite is compiled into a test executable with a fancy command-line interface provided by the Testo library.</p>
+<h2 id="main-features">Main features</h2>
+<ul>
+<li>Most tests can be defined by providing only a name and a function of type <code>unit -&gt; unit</code>.</li>
+<li>Tests can be assigned nested categories of arbitrary depth but are conveniently handled as a single flat list.</li>
+<li>Tests that are expected to fail (“XFAIL”) can be marked as such. This allows writing tests ahead of feature implementation or bug fixes.</li>
+<li>Supports output snapshots, i.e. capturing stdout or stderr from a test and comparing it with a reference file.</li>
+<li>Reviewing previous test outcomes can done without rerunning the tests.</li>
+<li>Provides various utilities for capturing stdout or stderr, and masking variable parts of test output such as temporary file paths.</li>
+<li>Support for tests that return Lwt promises.</li>
+</ul>
+<p>XFAIL outcomes and snapshot files are two features borrowed from Pytest that would have required massive changes in Alcotest and led to the creation of a new project.</p>
+<h2 id="should-i-use-testo">Should I use Testo?</h2>
+<p>Testo was designed to support older OCaml versions starting from 4.08 and to be maintained by the community of users. It is being used to test <a href="https://github.com/semgrep/semgrep">Semgrep</a> which has about 5000 OCaml tests, most of which were originally migrated from Alcotest. Check out the <a href="https://github.com/semgrep/testo/issues">known missing features</a> to see if anything critical to you is missing.</p>
+<h2 id="getting-started">Getting started</h2>
+<h3 id="install-the-testo-library">Install the <code>testo</code> library</h3>
+<p>🚧 <del>Install <code>testo</code> with Opam using <code>opam install testo</code></del> For now, we recommend using <code>testo</code> as a git submodule. Dune will pick it up and build it as part of your project like an ordinary library.</p>
+<h3 id="set-up-your-project">Set up your project</h3>
+<p>The folder <code>tests/snapshots/</code> will be used by Testo to store test snapshots to be tracked by your favorite version control system (git, …). Storing other test data under <code>tests/</code> is encouraged as long as you let Testo manage <code>tests/snapshots/</code>.</p>
+<h3 id="write-a-test-executable">Write a test executable</h3>
+<p>The test executable can be placed anywhere in your Dune project. We recommend having only one such program if possible and calling it <code>test</code>. In this tutorial, we’ll put it in the <code>tests/</code> folder.</p>
```

Comment:
The test executable sounds like you mean the test.exe. Maybe say "The test executable entry point module ..." so it's more obvious you talk about Test.ml?